### ISSUE TITLE
wasted @import font call

### ISSUE THREAD

**Issue by jdwillemse** [vestman/Select-or-Die#35]
Can you change the font to use a web-safe font instead of the google font?

The google font import makes an unnecessary call when the font is overridden in my project styles. When using bower it is not best practice to change anything in the packages you use. I import the package .css into my .css and then override specific rules. This doesn't work for the font @import. I can also copy the package .css to my project folder, but again this is not ideal as I'd essentially be forking the code locally.

This is a super small issue, but solving it would make this this already awesome plugin even awesomer!

**Comment by vestman** [comment]
I completely agree, @jdwillemse! Removed in the next version that will be pushed today or tomorrow.
